src/styles/style.css

```diff
@@ -0,0 +1,96 @@
+.card-wrapper {
+  max-width: 200px;
```

Comment:
you can set width for your card
it's  not a big deal here, cause width will be always static